components/src/components/hubspot-form/hubspot-form.tsx

```diff
@@ -46,6 +46,39 @@ export class HubspotForm {
             });
             this.isLoading = false;
             this.didLoad = true;
+
+            // Hidden form fields from HubSpot created forms causes gaps to appear in
+            // the form which is less then desireable. So we need to find the hidden
+            // form fields and then hide the parent <fieldset> for it. We will also need
+            // to populate those fields with the correct values.
+            waitForElementToExist(`#hubspotForm_${this.formId} form fieldset div`).then(() => {
+                const fieldSets = document.querySelectorAll(`#hubspotForm_${this.formId} form fieldset div[style*="display:none"]`);
+                fieldSets.forEach((fieldset) => {
+                    fieldset.parentElement.style.display = "none";
+                });
+
+                // Populate the UTM parameter fields.
+                const cookies = parseCookie();
+                const utmCookie: any = parseUTMCookieString(cookies["__utmzz"]);
+                const utmCampaign = utmCookie.utmccn || "(not set)";
+                const utmSource = utmCookie.utmcsr || "(direct)";
+                const utmMedium = utmCookie.utmcmd || "(none)";
+
+                const utmCampaignInput = document.querySelector<HTMLInputElement>(`input[name="last_utm_campaign"]`);
```

Comment:
Should these be `document.querySelector`, or could it be scoped more specifically to this component instance?

Reply:
👍